### PR TITLE
install linux script | provide link to support lifecycle doc

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

**Comment by bebound** [comment]
The Support lifecycle link is broken.

The whole page is outdated. The provided script is broken since Python3.10

Ref: https://github.com/Azure/azure-cli/issues/28791

**Comment by dbradish-microsoft** [comment]
@bebound, We cannot remove this zone pivot, but instead need an updated script. Until then, is this minor update more satisfactory?

**Comment by bebound** [comment]
Yes, the added version note should help clarify any misunderstandings.

File: docs-ref-conceptual/includes/cli-install-linux-script.md [hunk 1]
```diff
@@ -11,21 +11,24 @@ ms.custom: devx-track-azurecli, linux-related-content
...
-* [Python 3.6.x, 3.7.x or 3.8.x](https://www.python.org/downloads/).
+* [Python 3.8.x, 3.9.x, 3.10.x](https://www.python.org/downloads/) - For more information, see [Support lifecycle - Python dependency](../azure-cli-support-lifecycle.md#python-dependency).
 * [libffi](https://sourceware.org/libffi/)
 * [OpenSSL 1.0.2](https://www.openssl.org/source/)
 
 ## Install or update Azure CLI
 
-Both installing and updating the Azure CLI requires rerunning the install script.
+> [!IMPORTANT]
+> The install script only works on Python 3.8.x, 3.9.x, or 3.10.x. This install script does not work on Python 3.11.x or later versions.
```
Comment (bebound) [inline docs-ref-conceptual/includes/cli-install-linux-script.md:23]: The install script works from 3.6 to 3.10.

Comment (dbradish-microsoft) [reply docs-ref-conceptual/includes/cli-install-linux-script.md:23]: Thanks for the clarification, @bebound. We are removing all references to CentOS b/c of EOL, so I don't think we should mention EOL Python versions here for the same reason. Does this seem logical to you, or what customer value do you see for mentioning 3.6 and 3.7?

Comment (bebound) [reply docs-ref-conceptual/includes/cli-install-linux-script.md:23]: This is an old script, and I guess most of the customer are using them with old Python version. Removing 3.6 and 3.7 causes confusion.

I'm okay if they are removed.

**Pull request merged**